common/include/pcl/impl/point_types.hpp

```diff
@@ -960,7 +960,7 @@ namespace pcl
     }
 
     inline PointXYZRGBNormal (float _curvature = 0.f):
-        PointXYZRGBNormal (0.f, 0.f, 0.f, 0.f, 0.f, 0.f, 0.f, 0.f, 0.f, _curvature) {}
+        PointXYZRGBNormal (0.f, 0.f, 0, 0, 0, 0.f, 0.f, 0.f, 0.f, _curvature) {}
```

Comment:
Are you sure this is correct? Shouldn't it be 1-3 float, 4-6 unsigned, 7-9 float as well? Now 1-2 are float, 3-5 are int, 6-10 floats. :')

Reply:
```suggestion
        PointXYZRGBNormal (0.f, 0.f, 0.f, 0, 0, 0, 0.f, 0.f, 0.f, _curvature) {}
```
aaaaaaaargh